config/env/default.js

```diff
@@ -26,6 +26,17 @@ module.exports = {
   // sessionKey is set to the generic sessionId key used by PHP applications
   // for obsecurity reasons
   sessionKey: 'sessionId',
+  // Lusca CSRF config
+  csrf: {
+    csrf: false,
```

Comment:
Why is CSRF false by default?
Also, doesn't CSRF require updates on the frontend part to send a token?